app/src/main/java/com/example/dsublett/monsterfetch/models/DataModels.kt

```diff
@@ -2,4 +2,44 @@ package com.example.dsublett.monsterfetch.models
 
 // Data classes required to parse JSON response with Moshi
 data class DNDAPIResponse(val count: Int, val results: List<ResponseItem>)
+
 data class ResponseItem(val name: String, val url: String)
+
+/* For MonsterResponse, ClassResponse and SpellResponse, I am only parsing fields that can be
+ * represented as a String, Int, List<String> or List<Int>. Complex fields that would require
+ * creating more models are being ignored for now. Also fields such as _id, index and url are being
+ * ignored, since they are not being used in the app.
+ */
+
+data class MonsterResponse(
+    val name: String,
+    val type: String,
+    val subtype: String,
+    val alignment: String,
+    val armor_class: Int,
+    val hit_points: Int,
```

Comment:
Use Moshi's annotation `@Json(name = "hit_points") val hitPoints: Int` to maintain camelCase in your code base